components/UI/ReasoningFilterMenu/index.tsx

```diff
@@ -99,9 +114,9 @@ const ReasoningFilterMenu = ({ onChange }: ReasoningFilterMenuProps) => {
           disableTouchRipple
         >
           {
-            reasoningFilterMenuOptions.find(
-              (option) => option.value === selectedValue
-            )?.label
+            reasoningFilterMenuOptions.find((option) => {
```

Comment:
It would be a good improvement if we could hold this line in a variable somewhere in the component. If the PR is urgent, don't mind

Reply:
done